### DISCUSSION TITLE
Remove the empty button in the data component type 'table'

### DISCUSSION THREAD

**Discussion by Geni2** [pimcore/pimcore#16834] [Q&A / Support] [answered]
Hi all,
I'm using Pimcore 10.6

I have a data component type: table, and I wanted to know if it's possible to remove or hide the empty button, see the picture.
Also creating an event bubble to confirm the empty table would also be helpful.

Thank you in advance!

![table](https://github.com/pimcore/pimcore/assets/51850738/b0848bd2-7410-47b3-8b45-dd2f30bfb7f5)

**Comment by Geni2** [comment]
Hey @blankse , can you help me with this, Thanks.

**Reply by blankse** [reply, marked as answer]
@Geni2 You can hide it with css:

1. Create a src/Bundle/CustomAdminBundle/CustomAdminBundle.php file:
```
<?php

namespace App\Bundle\CustomAdminBundle;

use Pimcore\Extension\Bundle\AbstractPimcoreBundle;
use Pimcore\Extension\Bundle\PimcoreBundleAdminClassicInterface;
use Pimcore\Extension\Bundle\Traits\BundleAdminClassicTrait;

class CustomAdminBundle extends AbstractPimcoreBundle implements PimcoreBundleAdminClassicInterface
{
    use BundleAdminClassicTrait;

    public function getCssPaths(): array
    {
        return [
            '/bundles/customadmin/css/admin.css',
        ];
    }
}
```

2. Activate the bundle in config/bundles.php
```
<?php

return [
    \App\Bundle\CustomAdminBundle\CustomAdminBundle::class => ['all' => true],
];
```

3. Create the CSS file src/Bundle/CustomAdminBundle/public/css/admin.css
```
.object_field_type_table .x-toolbar a:last-child {
    display: none;
}
```

4. Execute asset:install
```
php bin/console assets:install
```

**Reply by Geni2** [reply]
Thank you for your response.
Just for clarification, also with Pimcore 10.6 I can use this approach?

**Reply by blankse** [reply]
Yes, in Pimcore 10.6 you must remove PimcoreBundleAdminClassicInterface/BundleAdminClassicTrait from my example

**Reply by Geni2** [reply]
Okay got it, many thanks @blankse
